packages/components/pages/f-checkout/stories/Checkout.stories.js

```diff
@@ -34,50 +34,10 @@ export const CheckoutComponent = () => ({
             getCustomerUrl: '/get-customer'
         };
     },
-    props: {
-        isLoggedIn: {
-            default: boolean('Is User Logged In', false)
-        },
-
-        serviceType: {
-            default: select('Service Type', propOptions.serviceTypeOptions, 'delivery')
-        },
-
-        locale: {
-            default: select('Locale', [locales.gb, locales.au, locales.nz], locales.gb)
-        },
-
-        isAsapAvailable: {
-            default: boolean('Is ASAP available', true)
-        },
-
-        getCheckoutOptions: {
-            default: select('Get Checkout Options', propOptions.getCheckoutOptions, null)
-        },
-
-        getBasketError: {
-            default: select('Get Basket Options', propOptions.getBasketOptions, null)
-        },
 
-        patchCheckoutError: {
-            default: select('Patch Checkout Errors', propOptions.patchCheckoutErrorOptions)
-        },
-
-        createGuestError: {
-            default: select('Create Guest Errors', propOptions.createGuestErrorOptions, null)
-        },
-
-        placeOrderError: {
-            default: select('Place Order Errors', propOptions.placeOrderErrorOptions)
-        },
-
-        fulfilmentTimeErrors: {
-            default: select('Fulfilment Time Options', propOptions.fulfilmentTimeErrors)
-        },
-
-        noteType: {
-            default: select('Note types', propOptions.noteTypeOptions, null)
-        }
+    props: Object.keys(argTypes),
+    beforeCreate () {
+        console.log(args, argTypes);
     },
 
```

Comment:
Do you still want this console.log?

Reply:
Noooo, I thought I found them all. Thank you!